pkg/testutil/tempdir.go

```diff
@@ -71,3 +71,8 @@ func (td *TempDir) Write(file string, content []byte) *TempDir {
 	}
 	return td
 }
+
+// Touch creates a file without content in the temporary directory.
+func (td *TempDir) Touch(file string) *TempDir {
```

Comment:
let's not add this here just because you're using it from migration code.

Reply:
Agreed. I imagine we will have other uses for this method, but this PR is not a good place to add it.